lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -889,8 +889,16 @@
     devices:
      - "/dev/sda:/dev/xvda:rwm"
     ports:
+     # Publish container port 9000 as host port 8080
      - "8080:9000"
+     # Publish container UDP port 9001 as host port 8081 on interface 127.0.0.1
      - "127.0.0.1:8081:9001/udp"
+     # Publish container port 9002 as a random host port
+     - "9002"
+     # Publish container port 9003 as a random host port in range 8000-8100
+     - "9003:8000-8100"
```

Comment:
I think these may be reversed order. Should be `8000-8100:9003`, right?

Reply:
Also think the comment can mention that it starts with lowest available one, not a random one, in that range. Maybe something like `# Publish container port 9003 to next available host port in range 8000-8100, going from lowest to highest range value`. I’m sure it could be worded better.

Reply:
Indeed! Looks like I got confused when switching back from the single-port variant to the two-ports(-ranges) variant...

And good point about "random", it is not random indeed. I wouldn't specify the exact behavior though, just that it uses a free port in that range.

Reply:
Sounds good, but it had the "random" word in the sentence and thought it might give a slightly incorrect impression.